patterns/patterns-sailfish-device-adaptation-pdx201.inc

```diff
@@ -15,7 +15,7 @@ Requires: libhybris-libwayland-egl
 Requires: bluebinder
 
 # Telephony
-Requires: ofono-ril-binder-plugin
+Requires: ofono-binder-plugin
```

Comment:
this will sadly break community builds on 4.3.0 (and all future versions until ofono-binder-plugin is released)..

Reply:
Hmm... is there a better way to do it? Besides, ofono-binder-plugin has already been merged and I'd assume that it could be pulled into into a community build too. Although I'd first wait until the dust settles, since it's a lot of new code.

Reply:
Yes it could be pulled into https://build.sailfishos.org/project/show/nemo:devel:hw:common, so all builds get it seamlessly.

Any other dependencies that can be dropped or need pulled-in? As now we get e.g. libgbinder from 4.3.0:hw:common, we could revert some of these [alternatively](https://github.com/mer-hybris/droid-hal-device/commit/3071d7d688d56bf14b99bef5d19ba7ddbf275aed#diff-65edf71f56b555c79b29729b8ce3bbcd8a7599be53cfea6bd08c6001370cb550)

Reply:
This new package obsoletes [ofono-ril-plugin](https://github.com/mer-hybris/ofono-ril-plugin) (in addition to [ofono-ril-binder-plugin](https://github.com/mer-hybris/ofono-ril-binder-plugin)).
And drops dependency on [libgrilio](https://github.com/sailfishos/libgrilio/) (required by ofono-ril-plugin).
No new dependencies.